Changelogs for Digital Twin Manager UI 1023.0.3 (#3981)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1023-2-0-enable-navigation-to-edit-asset-model-page-from-model-name-column.md

---
date: ""
title: "Enable Navigation to Edit Asset Model Page from Model Name Column"
product_area: "Application enablement & solutions"
change_type:
    - value: "change-2c7RdTdXo4"
      label: "Improvement"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-2296"
version: "1023.2.0"
---

> The Model Name column in the Asset Models screen is now interactive, allowing users to click a model name to view/edit asset model.

Reviewer: ```suggestion
The "Model name" column in the **Asset models** page is now interactive, allowing users to click a model name to view/edit the asset model details.
```

---
date: ""

> title: "Enable Navigation to Edit Asset Model Page from Model Name Column"

Reviewer: ```suggestion
title: "Navigation from model name to asset model details"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-2c7RdTdXo4"
      label: "Improvement"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"

> ticket: ""

Reviewer 2: ```suggestion
ticket: "CTM-2296"
```

version: "1023.2.0"
---

> The Model Name column in the Asset Models screen is now interactive,

Reviewer: ```suggestion
The "Model name" column in the **Asset models** page is now interactive.
```

> allowing users to click a model name to quickly navigate to the Edit

Reviewer: ```suggestion
On clicking a model name, you are now directed to the **Edit asset model** view where you can edit the model settings. 
```

> Asset Model page.

Reviewer: ```suggestion
```

> Asset Model page.

Reviewer 2: ```suggestion
The Model Name column in the Asset Models screen is now interactive, allowing users to click a model name to view/edit asset model.
```
